envy/lib/docker_manager/docker_manager.py

```diff
@@ -64,7 +64,11 @@ def create_container(self) -> ContainerManager:
         if self.get_container():
             raise ContainerExists()
 
-        # TODO: This likely should be done smarter: give the user a message if we need to download, etc
+        try:
+            self.docker_client.images.get(ENVY_CONFIG.get_base_image())
+        except docker.errors.ImageNotFound:
+            print("Pulling base image, this may take a while...")
+
         self.docker_client.images.pull(ENVY_CONFIG.get_base_image())
```

Comment:
Shouldn't this line be inside the `except` clause?

Reply:
We want to pull anyway, to ensure that we have the most up-to-date image. I don't think that we'll see an `ImageNotFound` error if the image exists but is out of date. This print is more of a "best effort", and we'll hope that update pulls won't be too bad.